src/util/async/AnyExecutionContext.hpp

```diff
@@ -66,9 +65,9 @@ class AnyExecutionContext {
     execute(SomeHandlerWithoutStopToken auto&& fn)
     {
         using RetType = std::decay_t<decltype(fn())>;
-        static_assert(not std::is_same_v<RetType, impl::Any>);
+        static_assert(not std::is_same_v<RetType, std::any>);
```

Comment:
Why not add this to the concept?

Reply:
No particular reason.. it was just coded up this way. We can change it but i'd prefer a separate issue for those.
Let's do it in #1346